doc/README.md

```diff
@@ -42,59 +73,165 @@ services:
+./dev_assets_watch_sass.sh
+```
+
+The command will start a container, which will monitor Sass directories for changes and compile them to CSS files when changes are detected.
+Note that if you have mounted the code to `/src/`, you need to restart the servers for these updates to take effect.
+
+3. After completing the changes and before committing them, run:
+
+```
+drone exec
+```
+
+4. Include the built production versions of JavaScript and CSS files (e.g. `css/main.css` and `css/main.css.map`) into the git commit with your changes.
+
+## Making changes to the asset packages
 
 The `assets_src/` contains folders, which are npm packages and should install compiled files to `assets/` as part of `npm install`.
 To manage these, you can use `./dev_assets_run_npm.sh`, which runs npm commands in a container.
 For example, to update `package.json` of the `translate-js`, run `./dev_assets_run_npm.sh translate-js update`.
 
-## SASS and JavaScript asset development
+## Running tests and updating translations
 
-Style sheets are developed with [SASS](https://sass-lang.com/), but that requires building the files to CSS before testing in the browser.
-This step is done as part of the Drone pipeline, but more interactive process can be started with `./dev_assets_watch_sass.sh` in the project root.
-The command will start a container, which will monitor SASS directories for changes and will compile them to CSS files when changes are detected.
-Note that if you have mounted the code to `/src/`, you need to restart the servers for these updates to take effect.
+Some parts are still easier to do without containers, tests and translations are such.
+Therefore they are run and updated outside the containers, within a virtual environment.
 
-After completing the changes to these types of assets, rerun the `drone exec` to build production versions of the CSS and JavaScript files, which should be added to the git for now (i.e. as long as there is no real CI/CD pipeline for A+).
+### Prerequisities
 
-## Running tests
+- Firefox browser installed
+- python3 installed
+- [Python module venv (or virtualenv)](https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/#installing-virtualenv) installed
 
-Before creating a pull-request, you should verify the code is still working against existing tests.
-Even better, you should write more tests, when possible!
-Nevertheless, you can do that on the host with the command `venv/bin/python manage.py test` (this assumes you did the the virtual environment installation from above).
+1. Create a virtual environment for a-plus and activate it (e.g. in the project root):
 
-## Selenium integration tests
+```
+python3 -m venv venv_aplus
+source venv_aplus/bin/activate
+```
 
-Selenium tests are designed to run against a certain database state and service ports.
+2. Install necessary requirements for development:
+
+```
+pip3 install --upgrade pip setuptools wheel
+pip3 install -r requirements.txt
```

Comment:
There could be a note to move to `a-plus` directory after creating and activating the virtual environment.

Reply:
Good point, I made that clearer now